test/test.js

```diff
@@ -22,4 +22,95 @@ describe('Array', function () {
 			assert.isTrue(callback.withArgs(3,2).calledOnce);
 		});
 	});
+
+	describe('#where', function() {
+		it('should return a new array filtered', function() {
+			//Setup
+			var array = [2, 5, 6];
+      var callback = sinon.stub();
+			callback.withArgs(2).returns(false);
+			callback.withArgs(5).returns(true);
+			callback.withArgs(6).returns(true);
+
+			//Execute
+			var newArray = array.where(callback);
+
+			//Compare
+			assert.equal(callback.callCount,3);
+			assert.isTrue(callback.withArgs(2).calledOnce);
+			assert.isTrue(callback.withArgs(5).calledOnce);
+			assert.isTrue(callback.withArgs(6).calledOnce);
+			assert.deepEqual(newArray, [5,6]);
+		})
+	});
+
+	describe('#any', function() {
+		it('should return true and call function 3 times', function() {
+			//Setup
+			var people = [
+				{ name: 'juan', age: 15 },
+				{ name: 'pablo', age: 16 },
+				{ name: 'topo', age: 18 },
+				{ name: 'pedro', age: 19 }
+	    ];
+			var callback = sinon.stub();
```

Comment:
Consider using a `function` that actually makes something and then [Spy it](http://sinonjs.org/docs/#sinonspy) instead of creating the **stub**